neurots/generate/tree.py

```diff
@@ -277,3 +283,6 @@ def next_point(self):
                 # TODO: Can the state be something else than "bifurcate" or "terminate" here?
                 # If not we could add an else statement to raise a warning or an exception if the
                 # value is anything else.
+            elif state == "continue":
```

Comment:
this is the main issue for imbalance: we add a child.first_point() at each bif, which makes the pathlength longer for branches with bifs, hence more likely to get more bifs later. We can either remove child.first_point(), or add a .next() to match the pathlengths.